google/resource_compute_instance_group_manager_test.go

```diff
@@ -201,43 +162,10 @@ func TestAccInstanceGroupManager_rollingUpdatePolicy(t *testing.T) {
 		Steps: []resource.TestStep{
 			resource.TestStep{
 				Config: testAccInstanceGroupManager_rollingUpdatePolicy(igm),
-				Check: resource.ComposeTestCheckFunc(
-					testAccCheckInstanceGroupManagerBetaExists(
-						"google_compute_instance_group_manager.igm-rolling-update-policy", &manager),
-					resource.TestCheckResourceAttr(
-						"google_compute_instance_group_manager.igm-rolling-update-policy", "update_strategy", "ROLLING_UPDATE"),
-					resource.TestCheckResourceAttr(
-						"google_compute_instance_group_manager.igm-rolling-update-policy", "rolling_update_policy.0.type", "PROACTIVE"),
-					resource.TestCheckResourceAttr(
-						"google_compute_instance_group_manager.igm-rolling-update-policy", "rolling_update_policy.0.minimal_action", "REPLACE"),
-					resource.TestCheckResourceAttr(
-						"google_compute_instance_group_manager.igm-rolling-update-policy", "rolling_update_policy.0.max_surge_percent", "50"),
-					resource.TestCheckResourceAttr(
-						"google_compute_instance_group_manager.igm-rolling-update-policy", "rolling_update_policy.0.max_unavailable_percent", "50"),
-					resource.TestCheckResourceAttr(
-						"google_compute_instance_group_manager.igm-rolling-update-policy", "rolling_update_policy.0.min_ready_sec", "20"),
-				),
 			},
+			// No import step because rolling updates are broken and the field will be removed in 2.0.0.
```

Comment:
Can we drop a TODO on this @danawillow to track its fix/removal?

Reply:
Done